### PR TITLE
Feature/upload photo

### PR DESCRIPTION
**Pull request by NightWanderer2004**
Add a user avatar upload feat. & interface for editing
Screenshot 2022-08-17 at 20 57 15
Screenshot 2022-08-17 at 21 06 43
Screenshot 2022-08-17 at 21 07 49

### CODE REVIEW

File: src/components/photoUpload/PhotoUpload.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import {
+  UPLOADING,
+  DONE,
+  ERROR,
+  IS_JPEG,
+  IS_PNG,
+} from "utils/photoUploadConsts";
+
+function PhotoUpload() {
+  const [isLoading, setIsLoading] = useState(false);
```
Comment (lyudad) [inline src/components/photoUpload/PhotoUpload.tsx]: type

File: src/components/photoUpload/PhotoUpload.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  const [imageUrl, setImageUrl] = useState<string>(
+    "https://pbs.twimg.com/profile_images/1541920820204797952/RaMNiqHx_400x400.jpg"
```
Comment (lyudad) [inline src/components/photoUpload/PhotoUpload.tsx]: never use url in the code. move to env or to constants

File: src/components/photoUpload/PhotoUpload.tsx (hunk 1 above)
Comment (NightWanderer2004) [reply src/components/photoUpload/PhotoUpload.tsx]: Added

File: src/components/photoUpload/PhotoUpload.tsx (hunk 2 above)
Comment (NightWanderer2004) [reply src/components/photoUpload/PhotoUpload.tsx]: Removed.
Sorry, this was for a test, forgot to remove at first commit

**Pull request merged**